src/main/kotlin/com/mparticle/kits/UrbanAirshipConfiguration.kt

```diff
@@ -95,6 +97,9 @@ class UrbanAirshipConfiguration(settings: Map<String, String>) {
         applicationKey = settings[KEY_APP_KEY]
         applicationSecret = settings[KEY_APP_SECRET]
         domain = settings[KEY_DOMAIN]
+        if (settings.containsKey(KEY_INITIAL_CONFIG_URL)) {
+            initialConfigUrl = settings[KEY_INITIAL_CONFIG_URL]
+        }
```

Comment:
Based on the configuration setting key change..

```suggestion
        if (settings.containsKey(KEY_CUSTOM_DOMAIN_PROXY_URL)) {
            initialConfigUrl = settings[KEY_CUSTOM_DOMAIN_PROXY_URL]
        }
```